DEVELOPING.md

```diff
@@ -0,0 +1,78 @@
+# Developing Liberty Tools for Visual Studio Code
+
+- [Build Liberty Tools for Visual Studio Code](#build-liberty-tools-for-visual-studio-code)
+- [Language Servers](#language-servers)
+  - [Build Liberty Config Language Server locally](#build-liberty-config-language-server-locally)
+  - [Build Eclipse LSP4Jakarta locally](#build-eclipse-lsp4jakarta-locally)
+- [Localization](#localization)
+  - [package.json](#packagejson)
+  - [Source code](#source-code)
+
+## Building Liberty Tools for Visual Studio Code
+
+1. Clone this repository: `git clone https://github.com/OpenLiberty/liberty-tools-vscode`
+2. Change to the `liberty-tools-vscode` directory: `cd liberty-tools-vscode`
+3. Run `npm install`
+4. Run `npm run build`
+5. Run `npm run compile`
+6. Run the extension in Debug and Run mode by selecting `Run Extension` or pressing `F5`
+
+   Alternatively, build a `.vsix` file and install the extension to Visual Studio Code with the command palette or Extensions activity bar.
+   - Run `vsce package` to generate the `liberty-tools-vscode-xxx.vsix` file
+   - Install the extension with the [command palette](https://docs.github.com/en/codespaces/codespaces-reference/using-the-vs-code-command-palette-in-codespaces#accessing-the-vs-code-command-palette):
+     - Select `Extensions: Install from VSIX...` and choose the generated `liberty-tools-vscode-xxx.vsix` file
+   - Install the extension with the Extensions activity bar:
+     - Navigate to the Extensions activity bar, or use the shortcut `Ctrl`-`Shift`-`X`
+     - Click the `...` dropdown menu and select `Install from VSIX` and choose the generated `liberty-tools-vscode-xxx.vsix` file
+
+Run `npm run build` to download the JARs for Liberty Config Language Server and Eclipse LSP4Jakarta that are consumed by the VS Code client.
```

Comment:
Why are we mentioning to run `npm run build` again here when it was already mentioned in step 4?

Reply:
Oh I must've made a edit that changed my original intent. I intended in this and the following sentence to segue into building the jars locally.